src/gym_duckietown/simulator.py

```diff
@@ -795,35 +795,26 @@ def _interpret_map(self, map_data: MapFormat1):
 
             # For each row in the grid
             for j, row in enumerate(tiles):
-
                 if len(row) != self.grid_width:
                     msg = "each row of tiles must have the same length"
                     raise InvalidMapException(msg, row=row)
 
                 # For each tile in this row
                 for i, tile in enumerate(row):
-                    tile = tile.strip()
-
-                    if tile == "empty":
+                    tile_type = tile.type
+                    if tile_type == "empty":
                         continue
 
                     directions = ["S", "E", "N", "W"]
                     default_orient = "E"
 
-                    if "/" in tile:
-                        kind, orient = tile.split("/")
-                        kind = kind.strip(" ")
-                        orient = orient.strip(" ")
-                        angle = directions.index(orient)
-
-                    elif "4" in tile:
+                    kind = tile_type
+                    orient = tile.orientation
+                    angle = directions.index(orient)
+                    if kind.startswith("4"):
                         kind = "4way"
                         angle = directions.index(default_orient)
```

Comment:
`tile.type` should already be `4way`, do not do things like `string.startswith("XYZ")`.